cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/TopicAnomalyFinder.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.detector;
+
+import com.linkedin.cruisecontrol.common.CruiseControlConfigurable;
+import java.util.Set;
+import org.apache.yetus.audience.InterfaceStability;
+
+
+/**
+ * The finder which checks topics against certain desired topic property to identify topic anomaly.
```

Comment:
Nit: Would it be more clear to say `violation of user-defined topic properties` rather than _certain desired topic property_?